### PR TITLE
[ja] add docs for owners-dependents

### LINKED ISSUE #38791: [ja] Translate concepts/overview/working-with-objects/owners-dependents/ into Japanese
**This is a Feature Request**

**What would you like to be added**

Translate https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/ into Japanese

**Why is this needed**

**Comments**

/language ja
/assign

### PR DESCRIPTION
**Pull request by FeLvi-zzz**
Add desrtiption of owners-dependents.
https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/

close #38791

### CODE REVIEW

File: content/ja/docs/concepts/overview/working-with-objects/owners-dependents.md
Comment (bells17) [inline content/ja/docs/concepts/overview/working-with-objects/owners-dependents.md:13]: `/ja/docs/concepts/overview/working-with-objects/labels/`

日本語ページがあるようです
https://kubernetes.io/ja/docs/concepts/overview/working-with-objects/labels/

File: content/ja/docs/concepts/overview/working-with-objects/owners-dependents.md
Comment (bells17) [inline content/ja/docs/concepts/overview/working-with-objects/owners-dependents.md:56]: `/ja/docs/concepts/architecture/garbage-collection/#cascading-deletion`

日本語ページがあるようです
https://kubernetes.io/ja/docs/concepts/architecture/garbage-collection/#cascading-deletion

**Review by bells17 — COMMENTED** [review]
2点コメントしました。
ご確認ください。

**Review by t-inu — COMMENTED** [review]
Japanese finalizer page will be available after merging PR https://github.com/kubernetes/website/pull/38768.
So, how about wait for the merge, and then reflect my suggestions?

**Review by t-inu — COMMENTED** [review]
I also reviewed the contents.

**Comment by FeLvi-zzz** [comment]
> Japanese finalizer page will be available after merging PR https://github.com/kubernetes/website/pull/38768.
So, how about wait for the merge, and then reflect my suggestions?

I reflected your suggestions, so I wait for it.

File: content/ja/docs/concepts/overview/working-with-objects/owners-dependents.md
Comment (t-inu) [reply content/ja/docs/concepts/overview/working-with-objects/owners-dependents.md:50]: I've checked the line 62 has been updated.
Are you going to update the line 50 later?

Comment (FeLvi-zzz) [reply content/ja/docs/concepts/overview/working-with-objects/owners-dependents.md:50]: Oh, I missed it...
I'm updating.

**Review by t-inu — APPROVED** [review]
/lgtm

**Comment by t-inu** [comment]
/hold
This PR contains a link to `/ja/docs/concepts/overview/working-with-objects/finalizers/`.
That is a new page and reviewing at https://github.com/kubernetes/website/pull/38768.
Feel free to unhold anyone when that PR is merged.

**Review by nasa9084 — COMMENTED** [review]
/assign
/approve

will `/unhold` after the relevant PR is merged

**Comment by t-inu** [comment]
PR https://github.com/kubernetes/website/pull/38768 was merged.
/unhold

**Pull request merged**
